package.json

```diff
@@ -7,7 +7,7 @@
     "generate-loopback-core": "node ./apidocs/describe-builtin-models.js",
     "prepublish": "npm run generate-loopback-core",
     "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run  --browsers PhantomJS",
-    "posttest": "eslint ."
+    "posttest": "npm run generate-loopback-core && eslint ."
```

Comment:
create a new task named "lint": "eslint .", then in update "posttest": "npm run generate-loopback-core && npm run lint". This way we can run the tasks separately if needs be in the future.